kwalitee/kwalitee.py

```diff
@@ -164,6 +174,14 @@ def _check_bullets(lines, **kwargs):
                     errors.append(("M121", i + j + 3))
                 else:
                     skipped.append(i + j + 1)
+                    if not _is_signature(indented):
```

Comment:
Signatures are separated with an empty line and are not indented to you don't need to check for them here.